udata_gouvfr/theme/templates/search.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'search.html' %}
+
+{% block extra_js %}
+<script type="text/javascript">
+  var _paq = _paq || [];
+
+  {% if datasets %}
+  var searchCount = {{ datasets.total }};
```

Comment:
not mandatory:
- `s/var/const/` (as it is now supported by every of our supported browser)
- I think you can write `{{ datasets.total | default(0) }}`